CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_hubspot_source v0.7.1 (May be a breaking change, TBD)
+
+## Updates
```

Comment:
should we include the link to this PR?

Reply:
Yes! I think I wrote this before creating the actual PR but will def make sure to link it!